Contributors.md

```diff
@@ -148,6 +149,7 @@ The following people have contributed to the SCAP Security Guide project
 * Sean O'Keeffe <seanokeeffe797@gmail.com>
 * Jiri Odehnal <jodehnal@redhat.com>
 * Ilya Okomin <ilya.okomin@oracle.com>
+* Juan Antonio Osorio <juan.osoriorobles@eu.equinix.com>
```

Comment:
Oz is already there, so https://github.com/ComplianceAsCode/content/blob/master/ssg/contributors.py should be probably updated, so he has just one entry in the file.